canonical-web-and-design/maas.io
# Replace latest-news script with npm version

## Done

Replaced the latest-news script with the npm version so it's easier to keep up to date if any changes are made

## QA

- Go to the homepage
- Scroll down to the "The latest MAAS news" section
- Check that the blog posts have loaded
- Check that the blog posts are the same as on maas.io

## Issue / Card

Fixes https://github.com/canonical-web-and-design/latest-news/issues/4
